### PR TITLE
Workspace Details view with list of applications in it

### PR DESCRIPTION
**Pull request by debsmita1**
## Fixes 
https://issues.redhat.com/browse/HAC-542

## Description
- Created a simple tabbed page containing the `Details` & `Applications` tabs
- the Applications tab lists all the applications if available otherwise the empty state is being shown, the most recently deployed application is listed first
- clicking on the application name will take the user to the Composition view page
- added tests

## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Gifs for design review 
Peek 2022-02-04 21-54

## How to test or reproduce?
Goto URL : https://prod.foo.redhat.com:1337/beta/hac/app-studio/workspaces?workspace=Purple_workspace

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/shared/components/timestamp/Timestamp.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  if (!dateTime.isValid(mdate)) {
+    return <div className="co-timestamp">-</div>;
+  }
+
+  if (simple) {
+    return <>{timeStamp}</>;
+  }
+
+  return (
+    <div className={classNames('co-timestamp co-icon-and-text', className)}>
```
Comment (rottencandy) [inline src/shared/components/timestamp/Timestamp.tsx:59]: These classes aren't defined either.

File: src/shared/components/simple-tab-nav/SimpleTabNav.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+};
+
+type SimpleTabNavProps = {
+  tabs: SimpleTab[];
+};
+
+const SimpleTabNav: React.FC<SimpleTabNavProps> = ({ tabs }) => {
+  const [selectedTab, setSelectedTab] = React.useState<string>(tabs[0].name);
+
+  const handleOnClickTab = (event, tabName: string) => {
```
Comment (divyanshiGupta) [inline src/shared/components/simple-tab-nav/SimpleTabNav.tsx:19]: nit: add event type

File: src/shared/components/table/Table.tsx [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+        NoDataEmptyMsg={NoDataEmptyMsg}
+        EmptyMsg={EmptyMsg}
+      >
+        {children}
+      </StatusBox>
+    </div>
+  );
+};
+
+export default Table;
```
Comment (divyanshiGupta) [inline src/shared/components/table/Table.tsx:322]: Please create separate files for all components.

**Review by divyanshiGupta — CHANGES_REQUESTED** [review]

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]
Don't think we need the tabbed view here for milestone 3. We just need the application list page. The workspace view is going to be built by infra team later on. @invincibleJai maybe we  should have updated the AC to remove that part after our conversation yesterday.

**Comment by debsmita1** [comment]
updated the branch name. so closing this and opened a new PR https://github.com/openshift/hac-dev/pull/39

**Pull request closed without merging**